### PR TITLE
Fix comment input ref

### LINKED ISSUE #764: Error in console v5
## I'm submitting a...

- [x] Bug report  
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ ] Feature request
- [ ] Documentation issue or request

## Context
1. Submitting a comment on the Structure page sidebar, the error is presented in the console.

2. Added HTML element and inserted text into it, error presented in the console. It is not visible every time.

### PR DESCRIPTION
**Pull request by lukasaric**
This PR fixes #764.
- First problem is fixed within this PR

### CODE REVIEW

**Comment by bubafinder** [comment]
I and @kjuej are still working on #764 issue, we will put all errors that are presented in the console in this issue.

File: client/components/repository/common/ActivityDiscussion.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
       @seen="setLastSeenComment"
       @unresolve="updateResolvement"
       v-bind="{ comments, unseenComments, showHeading, user }"
-      scroll-target="commentInput"
+      scroll-target="commentEditor"
```
Comment (underscope) [inline client/components/repository/common/ActivityDiscussion.vue]: Technically it is no longer an 'Editor'. Consider renaming inputContainer or some similar

File: client/components/common/tce-core/Discussion/index.vue [hunk 2]
```diff
@@ -97,7 +97,7 @@ export default {
     hasHiddenComments: vm => vm.commentsShownLimit < vm.commentsCount,
     isTextEditorEmpty: vm => !vm.comment.content?.trim(),
     discussion: vm => vm.$refs.discussion,
-    commentInput: vm => vm.$refs.commentInput,
+    commentEditor: vm => vm.$refs.commentEditor,
```
Comment (underscope) [inline client/components/common/tce-core/Discussion/index.vue]: If we are going to have it as computed lets have both or none. If not reused, we can just eliminate computed.

File: client/components/common/tce-core/Discussion/index.vue [hunk 3]
```diff
@@ -41,7 +41,7 @@
       :unseen-count="unseenComments.length"
       :user="user"
       class="mt-2" />
-    <div class="text-right">
+    <div ref="commentEditor" class="text-right">
```
Comment (underscope) [inline client/components/common/tce-core/Discussion/index.vue]: How does this fixes issue vs focusing on input ?

**Comment by underscope** [comment]
@malizeus 
Regarding 'Second problem and more are fixed within tce-jodit PR' please open up a PR with bumped tce-jodit version so it can be properly tested. Thx!

**Pull request merged**